Citizens/templates.yml terminatorgenisys

```diff
@@ -1246,6 +1246,38 @@ ironmanmkxv:
     uuid: 9b3e4ff2-136d-481c-8aa7-f858db93265b
     traitnames: spawned,type,inventory,owner,gravity,location
     name: sneaky
+    terminatorgenisys:
```

Comment:
incorrect spacing for terminatorgenisys: node
Is there a pic of the Terminator Genisys model? I can't find one by googling.